genx/make_deb.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
As already separated from setup script please move to debian_build folder.

Reply:
ok